app/assets/stylesheets/theme/main-content.scss

```diff
@@ -1,5 +1,8 @@
 .main-content {
+  position: relative;
+  z-index: 0;
```

Comment:
😬 I tend to lean on the layering generated by order-in-dom instead of `z-index` which can quickly escalate. Do you recall what caused this to be required?

Reply:
This is necessary to establish that the nav (and its contained dropdown) remain above the main content. Nav has `z-index: 1` and main content has `z-index: 0` as sibling selectors.